src/frontend/data/websocket.spec.ts

```diff
@@ -57,25 +58,25 @@ const ltiToken = {
 };
 
 describe('initVideoWebsocket', () => {
+  beforeEach(() => {
+    jest.resetAllMocks();
+  });
+
   afterEach(() => {
     fetchMock.restore();
     WS.clean();
   });
-  beforeEach(() => {
-    jest.resetAllMocks();
-  });
 
+  let initVideoWebsocket: any;
+  jest.isolateModules(() => {
+    initVideoWebsocket = require('./websocket').initVideoWebsocket;
+  });
```

Comment:
This is now outside the test because:
- it loads the file in a dedicated sandbox
- so it also loaded it dependencies in this sandbox, especially `useAuthenticatedUser` added in the PR
- `useAuthenticatedUser` will load `__mocks__/zustand.js`
- `__mocks__/zustand.js` declare and afterEach, but declaring an afterEach scope inside a test causes the test to fail